cron_trigger_autoupdate.py

```diff
@@ -1,5 +1,5 @@
 #!/usr/bin/env python
-# $ bash <(curl -fsS https://raw.githubusercontent.com/gilzow/source-operations/main/setup.sh) autoprsourceop
+# $ bash <(curl -fsS https://raw.githubusercontent.com/platformsh/source-operations/main/setup.sh) autoprsourceop
 # https://console.platform.sh/paul-gilzowatplatform-sh/rdj2cferlaluk/update/log/sqydnupxgbjqe
```

Comment:
So we want to keep such a reference here?

Reply:
We can get rid of line 2. For line 3, Chad and I had discussed including a link to a log entry on the event of an issue, and this is there to remind me of the URL structure. I can move that to an internal doc, and remove it if needed.